data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1951 @@
+			`You help the medical workers get the agents on transport vehicles, and one of the Arach Heliarchs supervising the operation hands you <payment>.`
+			`	"Avoided, many casualties were, thanks to your assistance in our joint efforts, Captain," she says. "Of this cooperativeness, inform my superiors, I shall."`
+			choice
+				`	"Thank you, but I didn't really do anything too impressive. It was just a simple transport mission."`
+				`	"There were a lot of injured people there. What happened there with the outlaws, exactly?"`
+					goto accident
+			`	"Nonsense. Looking for 'impressive' actions, we are not, when asking for assistance we are. Much more valued, your continuous assistance and cooperation is. The key to our Coalition's success, such assistance is."`
+				goto end
+			label accident
+			`	"A terrorist attack, it was. The valuable resources and equipment stockpiled there, they sought. Thanks to the agents' efforts though, robbed, the warehouses were not," she says, pointing to the injured Heliarchs being moved from your ship and into the land vehicles.`
+			label end
+			`	She bids you farewell and goes back to making sure the injured agents find their way to hospitals.`
+				accept
+
+mission "Heliarch Containment 3"
+	name "Informational Medicine"
+	description "While picking up <cargo> from <planet stopovers>, let loose the spy bots the Heliarchs gave you to act as surveillance for the planet."
+	cargo "medication" 15
+	to offer
+		has "Heliarch Containment 2: done"
+		not "Lunarium: Questions: done"
+	source "Ring of Power"
+	stopover "Secret Sky"
+	on offer
+		conversation
+			branch license
+				has "Heliarch License: done"
+			`Three Heliarch officers approach you as you're walking around the spaceport. "Greetings, Captain <last>. Interested, would you be, in accompanying us? A proposition, we have," the Saryd says.`
+			choice
+				`	"What's it about?"`
+				`	"Sure, lead the way."`
+					goto way
+				`	"Sorry, not up for taking a stroll right now."`
+					decline
+			`	"Inform you of that, we cannot," the Kimek says. "Not until accepted our offer, you have."`
+			choice
+				`	"Fine. Bring me to somewhere you can tell me."`
+				`	"That seems a bit sketchy, I'll pass."`
+					decline
+			label license
+			`Three Heliarch officers approach you as you're walking around the spaceport. They salute you, and the Saryd speaks. "Greetings, Captain <last>. Sent by Consul Plortub, we were."`
+			`	"A mission for you, he has, but to discuss it in private, he wishes to." The Kimek explains. "Follow us, would you?"`
+			label way
+			`	After a few dozen twists and turns through the restricted sections of the ring, they bring you to a room where an Arach is waiting, alongside a few other Heliarchs who are discussing something while pointing at a map of the Coalition projected over the table.`
+			`	"Ready to work with us, are you, Captain?" the Arach asks. "The one who asked for you, I am. Call me Consul Plortup, you may."`
```

Comment:
Is it Consul Plortup or Plortub? See below with Wiz's comment. Make sure you change all references in this mission(s?) to the proper one.